conf/backup_method

```diff
@@ -78,15 +83,21 @@ name="$3"
 size="$5"
 description="$6"
 
+saveIFS="$IFS"
+IFS=$';' array_repos=($repos)
+IFS="$saveIFS"
+
 case "$1" in
   need_mount)
-    do_need_mount "$work_dir" "$name" "$repo" "$size" "$description"
+    do_need_mount "$work_dir" "$name" "$(sanitize_repo "$array_repos[0]")" "$size" "$description"
     ;;
   backup)
-    do_backup "$work_dir" "$name" "$repo" "$size" "$description"
+    for repo in "${array_repos[@]}"; do
+      do_backup "$work_dir" "$name" "$(sanitize_repo "$repo")" "$size" "$description"
```

Comment:
Keep in mind if the first borg command to create the repo failed, the second won't be runned... (so it's not totally the same behavior than install twice the app)